.ci/setup-EAR.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Can we have a subdirectory for scripts? This top level directory is beginning to get crowded.

Reply:
I added it now. `os.ps1` and `os.sh` will need to move after the old jobs that use them are removed.